ckanext/csrf_filter/test_anti_csrf.py

```diff
@@ -70,7 +70,8 @@ def mock_objects(username=None):
     """
     anti_csrf.configure({
         'ckanext.csrf_filter.secret_key': 'secret',
-        'ckan.site_url': 'https://unit-test'})
+        'ckan.site_url': 'https://unit-test',
+        'ckanext.csrf_filter.exempt_rules': '["^/datatables/ajax/.*"]'})
```

Comment:
can you add another test with multiple rules and checking that they are loaded correctly into the exempt_rules list.